<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>随机生成器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditclassnum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditclassstunum">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelclassnum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>班级数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelclassstunum">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>学生人数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelresult123">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>51</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>结果:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGenerate">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditclassname">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>班级名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelresult">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>251</y>
      <width>511</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditInfo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>531</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>54</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOption">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOption">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
